tests/test_data_formats.py

```diff
@@ -42,6 +61,15 @@ def test_get_data_format_5_valid_data(self):
             encoded = DataFormats._get_data_format_5(x)
             assert encoded is not None
 
+    def test_get_data_format_e1_valid_data(self):
```

Comment:
You could add similar teses that were added for data format 6 in here https://github.com/ttu/ruuvitag-sensor/commit/62ee2f6c3d8a145e030c25462fb8547d89252cb3

You can check cases from official documentation: https://docs.ruuvi.com/communication/bluetooth-advertisements/data-format-e1